tests/valid_maintenance_contacts_phone.yml

```diff
@@ -0,0 +1,186 @@
+  - chat
+  - classroom-management
+  - museum
+  - music
+  - news
+  - ocr
+  - parallel-computing
+  - revision-control
+  - robotics
+  - scanning
+  - security
+  - sports
+
+categories:
+  - cloud-management
+
+usedBy:
+  - Comune di Firenze
+  - Comune di Roma
+
+roadmap: "https://designers.italia.it/roadmap/"
+
+developmentStatus: development
+
+softwareType: "standalone"
+
+intendedAudience:
+  onlyFor:
+    - cities
+    - health-services
+    - it-ag-agricolo
+  countries:
+    - it
+    - de
+  unsupportedCountries:     # Explicitly unsupported countries
+    - us
+
+description:
+  it:
+    localisedName: Medusa
+    genericName: "Text Editor"
+    shortDescription: "Un software veramente interessante."
+  eng:
+    localisedName: Medusa
+    genericName: "Text Editor"
+    shortDescription: "A really interesting software."
+    longDescription: >
+          Very long description of this software, also split
+          on multiple rows. You should note what the software
+          is and why one should need it. This is 158 characters.
+          Very long description of this software, also split
+          on multiple rows. You should note what the software
+          is and why one should need it. This is 316 characters.
+          Very long description of this software, also split
+          on multiple rows. You should note what the software
+          is and why one should need it. This is 474 characters.
+          Very long description of this software, also split
+          on multiple rows. You should note what the software
+          is and why one should need it. This is 632 characters.
+
+    documentation: "https://docs.developers.italia.it"
+    apiDocumentation: "https://developers.italia.it/it/api"
+
+    features:
+       - Very important feature
+       - Will run without a problem
+       - Has zero bugs
+       - Solves all the problems of the world
+    screenshots:
+       - tests/img/sshot1.png
+       - tests/img/sshot2.png
+       - tests/img/sshot3.png
+    videos:                             # Demo videos of the software
+       - https://www.youtube.com/watch?v=RaHmGbBOP84
+    awards:
+       - 1st Price Software of the year
+    freeTags:
+      - freeTag
+      - FreeoloTag
+
+legal:
+  license: AGPL-3.0-or-later        # SPDX expression of license
+  mainCopyrightOwner: City of Chicago
+  repoOwner: City of Chicago
+  authorsFile: tests/AUTHORS              # file listing copyright information
+
+maintenance:
+  type: "contract"
+
+  contractors:
+    - name: "Fornitore Privato SPA" # if maintainance is a contract
+      website: "https://developers.italia.it"
+      until: "2019-01-01"
+
+  contacts:
+    - name: Francesco Rossi
+      email: "francesco.rossi@comune.reggioemilia.it"
+      affiliation: Comune di Reggio Emilia
+      phone: 023113215112
+    - name: Dario Bianchi
+      email: "dario.bianchi@fornitore.it"
+      affiliation: Fornitore Privato S.P.A.
+      phone: +391624231322
```

Comment:
```suggestion
      # Test that a phone number with no spaces and a leading + is parsed correctly despite being potentially parsed as a number instead of a string.
      phone: +391624231322
```